{"name": "test_check_fz", "status": "failed", "statusDetails": {"message": "AssertionError: 设置盲区值1失败\nassert [2] == 1\n  +[2]\n  -1", "trace": "def test_check_fz():\n        for i in range(1,99):\n            cloud.fz = 1\n            cloud.send_device_datapoint()\n            time.sleep(60)\n            device_data = cloud.get_device_detail()\n            fz_data = jsonpath.jsonpath(device_data, '$...dataPoints[?(@.dataPointName == \"FZ\" )].dataPointReportedValue')\n            print(fz_data)\n>           assert fz_data == cloud.fz, '设置盲区值%s失败' % cloud.fz\nE           AssertionError: 设置盲区值1失败\nE           assert [2] == 1\nE             +[2]\nE             -1\n\ntest_send_dotapoint_check.py:32: AssertionError"}, "start": 1656925280687, "stop": 1656925341660, "uuid": "c7bb22a7-812b-4116-a2d1-ad4ef28f2295", "historyId": "31ce12057b51b2b2aa64411858c37bc5", "testCaseId": "9d060d588761426a3eb09cb001bbea6a", "fullName": "scripts.test_send_dotapoint_check#test_check_fz", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_send_dotapoint_check"}, {"name": "host", "value": "HONGCHA"}, {"name": "thread", "value": "9956-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_send_dotapoint_check"}]}